chaincode/abstore/java/build.gradle

```diff
@@ -25,7 +25,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'org.hyperledger.fabric-chaincode-java:fabric-chaincode-shim:2.0.+'
+    implementation 'org.hyperledger.fabric-chaincode-java:fabric-chaincode-shim:2.1.+'
```

Comment:
Doesn't matter too much, but I was thinking we would do 2.+ since master represents all 2.0 and beyond work.